<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NuevaVinculacion</class>
 <widget class="QDialog" name="NuevaVinculacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>606</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>710</width>
    <height>606</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Vinculación</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/prefijoNuevo/logo.png</normaloff>:/prefijoNuevo/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame{background-color:white;border-radius:10px}

QPushButton{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 15px;
letter-spacing: 0.02em;
color:white;
padding:8px;
background: #54BFC9;
border-radius: 8px;}

QPushButton:hover{background: #439CA8;
border-radius: 8px; }
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>691</width>
        <height>471</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame_2{
	
	background-color: rgb(255, 255, 255);
}

QLabel{
font-family: Roboto;
font-style: normal;
font-weight: 450;
font-size: 15px;
letter-spacing: 0.02em;
color: #6B7280;}


QLineEdit{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
border: 2px solid #F3F4F6;
border-radius: 8px;
color:#9CA3AF;
padding:10px}

QLineEdit::focus{
background: #FFFFFF;
border: 2px solid #344647;
border-radius: 8px;
}



/* COMBOBOX */

QComboBox{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
color:#9CA3AF;
border-radius:8px;
padding:10px;
combobox-popup: 0;
border: 2px solid #F3F4F6;

}


QComboBox::drop-down
{
    width: 20px;
    border-left-width: 1px;
	padding: 10px 10px 10px 20px;
 }


QComboBox QAbstractItemView {
	color:#9CA3AF;
	font-size:14px;
	background-color: #F3F4F6;
	selection-background-color:#344647;
	selection-color:#ffffff;
	outline: 0px;
   border: 1px solid #344647;
 border-radius:8px;
padding:10px
}



QComboBox::down-arrow
{
	image: url(:/menu/contraerabajo.png);
     width: 11px;
     height: 11px;
}

QComboBox::focus{
background: #FFFFFF;
border: 2px solid #344647;
border-radius: 8px;
}





/* PLANTEXT*/
QPlainTextEdit{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
border: 2px solid #F3F4F6;
border-radius: 8px;
color:#9CA3AF;
padding:10px}

QPlainTextEdit::focus{
background: #FFFFFF;
border: 2px solid #344647;
border-radius: 8px;
}


QCheckBox{font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
color:#9CA3AF}


/*SPINBOX*/
QSpinBox{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
color:#9CA3AF;
padding:8px;
border-radius:8px; }

QSpinBox::up-button {
min-width: 25px; min-height: 12px; background-color: #F3F4F6; 
border-radius: 6px;}

QSpinBox::down-button {
min-width: 25px; min-height: 12px; background-color: #F3F4F6; 
border-radius: 6px;}

QSpinBox::up-arrow{
	image: url(:/menu/contraerarriba.png);
     width: 11px;
     height: 11px;
}

QSpinBox::down-arrow{
	image: url(:/menu/contraerabajo.png);
     width:11px;
     height: 11px;}

QSpinBox::focus{
background: #FFFFFF;
border: 2px solid #344647;
border-radius: 8px;
}


QDateEdit{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
color:#9CA3AF;
padding:8px;
border-radius:8px; }

QDateEdit::up-button {
min-width: 25px; min-height: 12px; background-color: #F3F4F6; 
border-radius: 6px;}

QDateEdit::down-button {
min-width: 25px; min-height: 12px; background-color: #F3F4F6; 
border-radius: 6px;}

QDateEdit::up-arrow{
	image: url(:/menu/contraerarriba.png);
     width: 11px;
     height: 11px;
}

QDateEdit::down-arrow{
	image: url(:/menu/contraerabajo.png);
     width: 11px;
     height: 11px;}

QDateEdit::focus{
background: #FFFFFF;
border: 2px solid #344647;
border-radius: 8px;
}


</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="lbl5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>N° horas</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_nombreapellidos">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>36</y>
         <width>431</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="maxLength">
        <number>30</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>Ingrese nombre y apellidos</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Nombre y apellidos*</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
      <widget class="QLabel" name="lbl9">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>97</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Carrera*</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
      <widget class="QComboBox" name="cbox_carrera">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>120</y>
         <width>205</width>
         <height>40</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QLabel" name="label_95">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Periodo Academico*</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="fecha_inicio">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>220</y>
         <width>205</width>
         <height>40</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="displayFormat">
        <string>d-MM-yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lbl9_2">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>10</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Tipo identificación*</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_identificacion">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>205</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>ingrese identificacion</string>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::LogicalMoveStyle</enum>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_periodoacademico">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>205</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="maxLength">
        <number>30</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>Ingrese periodo acad...</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cbo_tipo_identificacion">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>36</y>
         <width>205</width>
         <height>40</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Cedula</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Carnet extranjeria</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pasaporte</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="lbl9_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>94</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Identificacion*</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_96">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>190</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Fecha inicio*</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="fecha_final">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>220</y>
         <width>205</width>
         <height>40</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="displayFormat">
        <string>d-MM-yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_97">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>190</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Fecha final</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spb_numerohoras">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_campoespecifico">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>310</y>
         <width>205</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="maxLength">
        <number>30</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>ingrese identificacion</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl9_5">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>280</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Campo especifico*</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
      <widget class="QLabel" name="lbl5_2">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>280</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Tutor Asignado*</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
      <widget class="QComboBox" name="cbo_tutor">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>310</y>
         <width>205</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lbl9_3">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>97</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Institución</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
      <widget class="QComboBox" name="cbo_institucion">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>120</y>
         <width>205</width>
         <height>40</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QComboBox" name="cbo_proyectos">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>400</y>
         <width>661</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>661</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>661</width>
         <height>40</height>
        </size>
       </property>
       <property name="maxVisibleItems">
        <number>10</number>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lbl5_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>370</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Proyecto Asignado*</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_codigoies">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>104</x>
         <y>310</y>
         <width>111</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="maxLength">
        <number>30</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>Cod. ies</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl5_4">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>280</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>N° Ies*</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btn_guardar">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>550</y>
        <width>195</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>195</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>40</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">/*  BOTON */

QPushButton{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 15px;
letter-spacing: 0.02em;
color:white;
padding:8px;
background: #3A4F50;
border-radius: 8px;}

QPushButton:hover{background: #344647;
border-radius: 8px; }
</string>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
      <property name="icon">
       <iconset resource="../image/source.qrc">
        <normaloff>:/menu/botonadd.png</normaloff>:/menu/botonadd.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>15</width>
        <height>21</height>
       </size>
      </property>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>711</width>
        <height>70</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>4455</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame_3{background: #3A4F50 ;border-top-left-radius: 10px;
border-top-right-radius: 10px;}

QLabel{

font-family: Roboto;
font-style: normal;
font-weight: 500;
font-size: 17px;
line-height: 40px;
letter-spacing: 0.02em;
color:#ffffff;

}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QToolButton" name="btn_closeadmi">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>10</y>
         <width>25</width>
         <height>25</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Cerrar</string>
       </property>
       <property name="styleSheet">
        <string notr="true">#btn_closeadmi{
font-family: Roboto;
font-weight: normal;
font-size: 14px;
background-color: #d5d5d8;
color: #6B7280;
border-radius:11px
}



#btn_closeadmi:hover{background:#ce4d3a ;
border-radius:11px; }


</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../image/source.qrc">
         <normaloff>:/menu/x.png</normaloff>:/menu/x.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>13</width>
         <height>13</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="axax">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Nueva Vinculación</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="QLabel" name="vinculacion_titulo_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Estado:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="QLabel" name="vinculacion_titulo">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>10</y>
         <width>391</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Nueva Afiliación</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="QLabel" name="vinculacion_estado">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Pendiente</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>line_nombreapellidos</tabstop>
  <tabstop>cbo_tipo_identificacion</tabstop>
  <tabstop>line_identificacion</tabstop>
  <tabstop>cbox_carrera</tabstop>
  <tabstop>cbo_institucion</tabstop>
  <tabstop>line_periodoacademico</tabstop>
  <tabstop>fecha_inicio</tabstop>
  <tabstop>fecha_final</tabstop>
  <tabstop>spb_numerohoras</tabstop>
  <tabstop>line_codigoies</tabstop>
  <tabstop>line_campoespecifico</tabstop>
  <tabstop>cbo_tutor</tabstop>
  <tabstop>cbo_proyectos</tabstop>
 </tabstops>
 <resources>
  <include location="../image/source.qrc"/>
 </resources>
 <connections/>
</ui>
